routes/expressions.js

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import verifyToken from './verifyToken';
+import { User, Expression, UserExpression } from '../models';
+
+const router = express.Router();
+
+const addExpression = async (request, response) => {
+  try {
+    const { value } = request.body;
+    const { userId } = request.user;
```

Comment:
You don't need to change this but i want to show you a trick. You can write these two lines as `const { body: { value }, user: { userId } } = request;`